scripts/run_clustering.py

```diff
@@ -1,36 +1,48 @@
 import mlflow
-from os.path import join
 from hyperopt import fmin, tpe, hp
 from hyperopt.pyll import scope
 from s2and.extentions.utils import load_dataset
 from s2and.extentions.clustering_models import Clusterer
-from s2and.extentions.featurization.utils import featurizing_function
 from s2and.extentions.clustering_objective import Objective
 from s2and.utils.configs import load_configurations
 from s2and.utils.mlflow import get_or_create_experiment
+from s2and import logger
+
 
 if __name__ == "__main__":
 
     cfg = load_configurations('configs/clusterer_conf.yml')
     datasets = [load_dataset(dataset_name) for dataset_name in cfg.datasets]
-    clusterers = [Clusterer(combined_classifier=cfg.model_source,
-                            dataset_name=dataset_name,
-                            featurization_function=featurizing_function,
-                            default_embeddings=cfg.default_embeddings,
-                            embeddings_path=join(cfg.external_embeddings_dir, dataset_name, f'{dataset_name}_embeddings.json'),
-                            clusterer='dbscan') for dataset_name in cfg.datasets]
+    clusterers = [
+        Clusterer(
+            combined_classifier=cfg.model_source,
+            dataset_name=dataset_name,
+            features=cfg.features,
+            embeddings_dir=cfg.external_embeddings_dir,
+            clusterer='dbscan'
+        ) for dataset_name in cfg.datasets
+    ]
 
     objective = Objective(dataset_names=cfg.datasets, datasets=datasets, clusterers=clusterers)
-    search_space = {'eps' : hp.uniform('eps', 0, 1), 'min_samples' : scope.int(hp.quniform('min_samples', 1, 5, q=1))}
+    search_space = {
+        'eps': hp.uniform('eps', 0, 1),
+        'min_samples': scope.int(hp.quniform('min_samples', 1, 5, q=1))
+    }
```

Comment:
the parameters here can be loaded by a config file.

Reply:
Random init again, hyperparam opt follows.